pkg/resource/deploy/step_generator.go

```diff
@@ -119,13 +119,14 @@ func (sg *stepGenerator) GenerateReadSteps(event ReadResourceEvent) ([]Step, res
 	}, nil
 }
 
-// GenerateSteps produces one or more steps required to achieve the goal state
```

Comment:
Are these comments being automatically reflowed?

Reply:
I'm manually triggering reflow when i encounter a comment that i'm needing to examine and/or potentially update.  I'm using the popular "rewrap" vscode plugin for this.

Reply:
Would you mind leaving these as-is unless you need to change them? The diffs are distracting.

Reply:
I can... though i find the inconsistency in comments distracting myself as i go read them.  I do prefer to just fix them up as a broken-window sort of thing.  I can try to do those in separate PRs though.

Reply:
> I'm using the popular "rewrap" vscode plugin for this.

JFWIW - I think you have it set to a different width than others - as you often submit changes that wrap at a different column than what most others wrap comments at.  Would be good to have a standard on what wrap column width we use across codebases.